themes/bootstrap3/templates/Helpers/pagination.phtml

```diff
@@ -41,7 +41,7 @@
               [
                 'params' => $this->params,
                 'current' => $this->current,
-                'currentActive' => true,
+                'currentActive' => false,
```

Comment:
I think you can remove this; I don't see `currentActive` being used anywhere else after the refactoring.

Reply:
Oops, fixed.